build.proj

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="4.0" DefaultTargets="Test;Chocolatey" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Compile;Test;Chocolatey" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Is there a particular reason why you're explicitly calling `Compile` here instead of the `DependsOnTargets` attribute below